platform/commonUI/edit/src/services/TransactionService.js

```diff
@@ -37,10 +37,8 @@ define(
         function TransactionService($q, $log) {
             this.$q = $q;
             this.$log = $log;
-            this.transaction = false;
-
-            this.onCommits = [];
-            this.onCancels = [];
+            this.transaction = undefined;
```

Comment:
I'd be inclined to do away with this variable and just use `this.transactionStack[this.transactionStack.length() - 1]`, it's one less thing to maintain and it can potentially be inconsistent with the state of the stack